main/main.go

```diff
@@ -132,10 +140,7 @@ func main() {
 		&cfg,
 	)
 	if err != nil {
-		logger.Error(
-			"Failed to create app request network",
-			zap.Error(err),
-		)
+		logger.Fatal("Failed to create app request network", zap.Error(err))
 		panic(err)
 	}
 
```

Comment:
not part of this PR, but what hapepns when the function callback returns an error, I'm not sure where it gets handled, and how it's handled.

Reply:
Which function callback do you mean?

Reply:
the health checker's check function https://github.com/ava-labs/awm-relayer/pull/327/files/6dc2441bb03dc49a9e1be1f5c768c913dda73254#diff-327181d0a8c5e6b164561d7910f4eeffd41442d55b2a2788fda2aa2692f17ec0R163

Reply:
It returns a `503 (Service Unavailable)` HTTP response. https://github.com/alexliesenfeld/health/blob/main/examples/showcase/main.go